staging/src/k8s.io/code-generator/cmd/validation-gen/validation.go

```diff
@@ -145,26 +150,24 @@ func (g *genValidations) Init(c *generator.Context, w io.Writer) error {
 			"fieldPath":   c.Universe.Type(fieldPathType),
 			"fmtErrorf":   c.Universe.Type(errorfType),
 		}
-		if isTypeWith(t, []string{"TypeMeta"}) {
```

Comment:
Also thanks for getting rid of TypeMeta here.

Reply:
Yeah, we already filtered for it before, so this was redundant